packages/enzyme-utilities/src/mount.ts

```diff
@@ -0,0 +1,12 @@
+import {ReactElement} from 'react';
+import {mount as enzymeMount, ReactWrapper, MountRendererProps} from 'enzyme';
+import {addMountedWrapper} from './wrappers';
```

Comment:
Note: by making the `wrappers` file separate from `mount`, we facilitate testable code via jest mocks: https://github.com/Shopify/quilt/pull/127/files#diff-b57d56e2c24aea42642caf55dc0610b9R8

Reply:
Not usually a great reason to do this kind of separation